api/internal/git/repospec_test.go

```diff
@@ -285,11 +287,104 @@ func TestNewRepoSpecFromUrl_Smoke(t *testing.T) {
 				GitSuffix: ".git",
 			},
 		},
+		{
+			name:      "t15",
+			input:     "https://github.com/kubernetes-sigs/kustomize//examples/multibases/dev/?submodules=0&ref=v1.0.6&timeout=300",
```

Comment:
There's a comment in L117 above that says

> // No tests for submodules and timeout as the expectations are hard-coded
> 	// to the defaults for compactness.

Does this test change that? I'm wondering if we should remove that comment now.

Reply:
The comment is true. I copied this test from elsewhere and left the query params, but it is true that they were "inactive" in the test. I removed them.